test/tests/functional/pbs_user_reliability.py

```diff
@@ -80,15 +80,17 @@ def test_create_resv_from_job_using_runjob_hook(self):
         jid = self.server.submit(job)
```

Comment:
FYI (above), you don't need setUp() if the only thing you do is call the super's setUp()